shared/teams/channel/index.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react'
+import * as Kb from '../../common-adapters'
+import * as Types from '../../constants/types/teams'
+import * as ChatTypes from '../../constants/types/chat2'
+import * as Styles from '../../styles'
+import ChannelTabs from './tabs/container'
+import {TabKey} from './tabs'
+import {Row} from './rows'
+import renderRow from './rows/render'
+export type Sections = Array<{data: Array<Row>; header?: Row; key: string}>
+
+export type TabProps = {
+  selectedTab: TabKey
+  setSelectedTab: (t: TabKey) => void
+}
+
+export type Props = {
+  teamID: Types.TeamID
+  conversationIDKey: ChatTypes.ConversationIDKey
+  sections: Sections
+}
+
+const Channel = (props: Props & TabProps) => {
+  const {teamID, conversationIDKey, selectedTab, setSelectedTab} = props
+  const renderItem = ({item}: {item: Row}) => {
+    switch (item.type) {
+      case 'tabs':
+        return (
+          <ChannelTabs
+            teamID={teamID}
+            conversationIDKey={conversationIDKey}
+            selectedTab={selectedTab}
+            setSelectedTab={setSelectedTab}
+          />
+        )
+      case 'settings':
+      case 'header':
+      case 'divider':
+      case 'member':
+      case 'bot':
+      case 'bot-add':
+      case 'loading':
+        return renderRow(item, teamID, conversationIDKey)
+      default: {
+        throw new Error(`Impossible case encountered in channel page list: ${item}`)
+      }
+    }
+  }
+
+  const renderSectionHeader = ({section}) => (section.header ? renderItem({item: section.header}) : null)
+
+  const popupAnchor = React.useRef<React.Component<any>>(null)
```

Comment:
This was my way of launching the selection popup that didn't work. I don't think you want it here.